reviews/tuf-graduation.md

```diff
@@ -0,0 +1,65 @@
+# TUF Graduation Application
+
+The Update Framework (TUF) helps developers maintain the security of a software update system, even against attackers that compromise some repository or signing keys. TUF provides a flexible framework and specification that developers can adopt into any software update system.
+
+## TUF fulfills all the incubating and graduation criteria:
+
+### Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+
+* TUF is used in production by Cloudflare, Datadog, DigitalOcean, Docker, IBM, Kolide, LEAP, Microsoft, RedHat, VMware, and many others.   [Uptane](https://uptane.github.io/), the automotive variant of TUF, is sold by major automotive vendors and is being adopted by automakers with over 1/3 of the cars on US roads in their forthcoming cars.  Uptane is included as part of [Automotive Grade Linux](https://www.automotivelinux.org/), another Linux Foundation project.  
```

Comment:
Could you please add some additional detail on at least three of these production use cases to assist the TOC in evaluating the "quality and scope" aspect of this requirement. Details like the exact product/service (as opposed to just the company name), what exactly TUF is used for in that product/service, the size of the largest deployments in which TUF is used in those services (in terms of infrastructure size, usage/user volume, transaction volume, etc). And any other info that you think will assist the TOC in gauging "scope and quality".

Reply:
The linked ADOPTIONS.md refers to articles and webpages that have what information we have (which is the public blog posts, etc. from the adopting companies talking about their use).  Notary is used in many of the cloud native scenarios.  In the past, when we've asked for data to analyze proposed changes, etc. it is anonymized for privacy reasons and there are multiple rounds of review, sometimes taking months to get data.  (See: https://www.usenix.org/system/files/conference/atc17/atc17-kuppusamy.pdf and https://www.usenix.org/system/files/conference/nsdi16/nsdi16-paper-kuppusamy.pdf, for example.)   

I don't have permission to speak on behalf of any of the adopters and I don't have the level of data you are asking for in this request.  I'm not sure how to proceed.

Reply:
For the Notary production use cases I can fill in some details.

- Docker. We run Notary alongside the Docker public registry with public availability, and have done for some years. All the official library images are signed, as are images we use as part of our products, for example the Docker Enterprise images that we ship, and many other internal and external projects. We also ship Notary as part of the commercial Docker Trusted Registry product which is part of Docker Enterprise.
- IBM. IBM have run Notary for providing content trust on IBM container registry, on their public cloud https://console.bluemix.net/docs/services/Registry/registry_trusted_content.html#registry_trustedcontent
- Microsoft. Azure container registry recently shipped TUF support (I presume this is Notary based) https://docs.microsoft.com/en-us/azure/container-registry/container-registry-content-trust
- Harbor, now a CNCF incubator project, includes Notary support.
- Quay, the CoreOS/Red Hat container registry supports Notary.

Reply:
There is also now an [AWS implementation](https://github.com/awslabs/tough)